uctMaths/competition/interface/printer_entry.html

```diff
@@ -207,4 +220,4 @@ <h2>Registered Invigilators</h2>
         </td>
     </tr>
 </table>
-<br><br><br><br>
+<pdf:nextpage />
```

Comment:
What is this for?

Reply:
Is it necessary to add the <br>s back in? They cause an extra blank page to appear when a school has a lot of entries